test/autorest/paginggroup/zz_generated_models.go

```diff
@@ -111,16 +111,16 @@ type PagingGetNullNextLinkNamePagesOptions struct {
 
 // PagingGetOdataMultiplePagesOptions contains the optional parameters for the Paging.GetOdataMultiplePages method.
 type PagingGetOdataMultiplePagesOptions struct {
-	ClientRequestId *string
+	ClientRequestID *string
 	// Sets the maximum number of items to return in the response.
 	Maxresults *int32
 	// Sets the maximum time that the server can spend processing the request, in seconds. The default is 30 seconds.
 	Timeout *int32
 }
 
-// PagingGetPagingModelWithItemNameWithXmsClientNameOptions contains the optional parameters for the Paging.GetPagingModelWithItemNameWithXmsClientName
+// PagingGetPagingModelWithItemNameWithXMSClientNameOptions contains the optional parameters for the Paging.GetPagingModelWithItemNameWithXMSClientName
```

Comment:
Should "XMS" be removed here?

Reply:
I'm thinking we should remove the code to remove the "XMS" altogether.  I don't really understand why the rule was put in place in track 1 and none of the other languages do this.

Reply:
Or we remove it everywhere (leaving it in does produce some ugly code).

Reply:
So I was thinking about this more...since XMS is part of the header name I actually do think we should leave it everywhere. Why are we determining what parts of a header name stay and which dont? Seems weird to just do it for this case.

Reply:
I tend to agree, however if you look at the swagger for blob storage, almost all of their response headers are "x-ms-" so the fields in the response envelopes get really ugly (I tried this yesterday, the codegen is not so great).

If we're removing it for fields in response envelopes then IMO we should be removing it everywhere.

Reply:
Well I think that the right thing this to not remove this anywhere, it's part of the name of these headers, fields, etc. And, as you said earlier, none of the other languages do this either. Probably for this same reasoning, (bad analogy coming) but it's like we decide we dont like the word "virtual" in any of our headers or fields, why are we deciding what part of the header name is relevant and which isnt?

Reply:
For here, "XMS" should not be removed.

I think the only place where it's applicable (we can discuss in scrum) is for header responses.